spec/models/date_range_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+RSpec.describe DateRange do
+
+  describe '#dates' do
```

Comment:
Since this test targets a method on `Conference`, rather then on `DateRange`, I think this would be a better fit for the `Conference` model specs.